Program.cs

```diff
@@ -40,12 +40,40 @@
                         }
                     };
                 });
+builder.Services.AddAuthorization(options =>
+{
+    options.AddPolicy("IDSCReferenceTokenView", policy =>
+        policy.RequireAssertion(context  =>
+            context.User.HasClaim(c =>
+                c.Type is "identity-server-config" or "identity-server-config.reference-token" or "identity-server-config.reference-token:view")));
```

Comment:
Does this do what you expect it to do? I can't remember or being a C# keyword. Perhaps we could write some unit tests for these policies so we're sure that they do what we expect them to do.

Reply:
Yes, it does work as expected. These are logical patterns: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#logical-patterns